### ISSUE TITLE
Downtime pop-up appearing empty when hovering

### ISSUE THREAD

**Issue by Henley74** [centreon/centreon#216]
I would like to know if any fix was ever found to this bug (I think it's very old, it seems it never got patched)

When I add a downtime to a server, the downtime pop-up appears empty when hovered. It's kind of bothering, it seems to happen randomly. I haven't yet identified the conditions for this bug to happen
Was there any fix ever found to this problem ?

![72384128-1559b000-371d-11ea-8d4e-7e0080eb1051](https://user-images.githubusercontent.com/18121144/202201600-35155aa4-f63d-4e6a-b8ae-340adaf3d80d.png)

*Version : 19.10*

**Comment by ponchoh** [comment]
# 👋 @Henley74 

Have you considered an upgrade? 🚀 

https://docs.centreon.com/docs/upgrade/upgrade-from-19-10/
https://docs.centreon.com/docs/releases/lifecycle/

![image](https://user-images.githubusercontent.com/85764737/202281362-4254e190-86a2-4c23-88db-18e125911d11.png)

or under the deprecated page

![image](https://user-images.githubusercontent.com/85764737/202281635-279cf2c3-0123-413d-ace9-ba50724a2a62.png)

currently at 22.10

## 🤞

**Comment by ponchoh** [comment]
Hi @Henley74, that is a very good questions. A few changes have occurred since then. Can you share what have you tried to search for the changes so we do not duplicate work? In any case it would be highly recommended to upgrade as a whole instead of cherry picking.

**Comment by ponchoh** [comment]
I see, thank you. This is something I found. 🔎

# 👉 fd21fb70cbc445b152dcc8493575f1a7be68a6d2 👈

> fix(ui): fix ack & downtime tooltips on hover ([#9064](https://github.com/centreon/centreon-archived/pull/9064))

I hope it can help you. 😊

```shell
git log --after=2019-10-01 --before=2021-12-01 --all --grep='downtime.*hover'
commit fd21fb70cbc445b152dcc8493575f1a7be68a6d2
Author: Kevin Duret <kduret@centreon.com>
Date:   Thu Oct 1 17:46:56 2020 +0200

    fix(ui): fix ack & downtime tooltips on hover (#9064)
```

👍 best of luck! 🍀

**Comment by Henley74** [comment]
Thank you very much for your answer ! I never found this commit before.
The problem is : I can't edit the typescript files without recompiling right? (the problem is that I think I can't edit the source files directly, because I do not find them on the server (no /src/ folder and no .tsx file neither, anywhere)
